Code/Tools/AssetProcessor/native/ui/BuilderDataItem.cpp

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#include <native/ui/BuilderDataItem.h>
+
+namespace AssetProcessor
+{
+    const AZStd::string jobTypeDisplayNames[] = { "Analysis Jobs", "Processing Jobs" };
+    const AZStd::string invalidJobTypeDisplayName{ "Invalid Job Type" };
```

Comment:
you can use a AZStd::array just create the AZstd::string the other option is to just declare AZStd::string into this array.  
```suggestion
    static constexpr AZStd::array jobTypeDisplayNames{ "Analysis Jobs", "Processing Jobs" };
    static constexpr AZStd::array invalidJobTypeDisplayName{ "Invalid Job Type" };
```